src/components/charts/SavedValues/SavedValuesList.tsx

```diff
@@ -0,0 +1,263 @@
+import React, { useState, useEffect } from 'react'
+import { useParams, useHistory, RouteComponentProps } from 'react-router'
+import { DOCUMENTATION, URLS } from '../../../config'
+import emptyCustomChart from '../../../assets/img/app-not-configured.png'
+import { ReactComponent as Upload } from '../../../assets/icons/ic-arrow-line-up.svg'
+import { ReactComponent as Search } from '../../../assets/icons/ic-search.svg'
+import { ReactComponent as Clear } from '../../../assets/icons/ic-error.svg'
+import { ReactComponent as File } from '../../../assets/icons/ic-file-text.svg'
+import { ReactComponent as Edit } from '../../../assets/icons/ic-pencil.svg'
+import { ReactComponent as Delete } from '../../../assets/icons/ic-delete-interactive.svg'
+import { BreadCrumb, ErrorScreenManager, Progressing, showError, useBreadcrumb } from '../../common'
+import { SavedValueListResponse, SavedValueType } from './types'
+import EmptyState from '../../EmptyState/EmptyState'
+import { deleteChartValues, getChartValuesTemplateList } from '../charts.service'
+import './savedValues.scss'
+import PageHeader from '../../common/header/PageHeader'
+import { toast } from 'react-toastify'
+
+export default function SavedValuesList() {
+    const history: RouteComponentProps['history'] = useHistory()
+    const { chartId } = useParams<{ chartId }>()
+    const [loader, setLoader] = useState(false)
+    const [searchApplied, setSearchApplied] = useState(false)
+    const [searchText, setSearchText] = useState('')
+    const [savedValueList, setSavedValueList] = useState<SavedValueType[]>([])
+    const [filteredSavedValueList, setFilteredSavedValueList] = useState<SavedValueType[]>([])
+    const [errorStatusCode, setErrorStatusCode] = useState(0)
+    const [noResults, setNoResults] = useState(false)
+
+    useEffect(() => {
+        getData()
+    }, [])
+
+    const getData = (): void => {
+        setLoader(true)
+        getChartValuesTemplateList(chartId)
+            .then((response: SavedValueListResponse) => {
+                let list = response.result || []
+                list = list
```

Comment:
Suggestion: we can always combine these 2 line something like -> `const list = (response.result || []).map(.......)`

Reply:
done

Reply:
done